vignettes/using-based-on.Rmd

```diff
@@ -138,14 +137,12 @@ log_df <- run_log(MODEL_DIR)
 log_df
 ```
 
-## Filtering tags example
-
-Among other things, the run log contains the tags that have been assigned to each model. Here we use `purrr::map_lgl` to filter the run log to only the final model.
+Among other things, the run log contains any descriptions that have been assigned to each model. Here we use `filter` and `pull` to get the path to the final model.
```

Comment:
nit: consider using `filter()` rather than `filter` and `pull()` rather than `pull` to match usage elsewhere.